src/App.tsx

```diff
@@ -1,73 +1,40 @@
-import React from 'react';
 import './App.scss';
+
+import React, { useMemo, useState } from 'react';
+
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { DropDownMenu } from './Components/DropDownMenu';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [chosenPerson, setChosenPerson] = useState<Person | null>(null);
+  const [appliedQuery, setAppliedQuery] = useState('');
```

Comment:
Rename `appliedQuery `to `searchQuery `for clearer intent. This enhances readability and aligns better with related variables in your codebase.